homeassistant/loader.py

```diff
@@ -52,7 +55,21 @@
     "You are using a custom integration %s which has not "
     "been tested by Home Assistant. This component might "
     "cause stability problems, be sure to disable it if you "
-    "experience issues with Home Assistant."
+    "experience issues with Home Assistant"
+)
+CUSTOM_WARNING_VERSION_MISSING = (
+    "No 'version' key in the manifest file for "
+    "custom integration '%s'. This will cause a "
+    "future version of Home Assistant to block "
+    "this custom integration. "
+    "Please report this to the maintainer of '%s'"
```

Comment:
```suggestion
    "custom integration '%s'. This will not be "
    "allowed in a future version of Home "
    "Assistant. Please report this to the "
    "maintainer of '%s'"
```